src/main/webapp/profile.js

```diff
@@ -60,14 +60,28 @@ function createProfileDiv(user) {
     profileEmail.innerHTML = user.email;
 
     const profileTopics = document.createElement('p');
+    var isStudent;
     // Students and tutors have different properties, check to see what kind of profile we're displaying
     if (user.learning != undefined) {   // User is a student
```

Comment:
I think the login status now returns a "role" property which specifies if the user is a "student" or a "tutor". Not sure if that would be easier to use here.

Reply:
That's much easier, I add that in a new commit